optimum/quanto/quantize.py

```diff
@@ -133,12 +133,11 @@ def move_tensor(t, device):
             setattr(m, name, torch.nn.Parameter(move_tensor(param, "cpu")))
         for name, param in m.named_buffers(recurse=False):
             setattr(m, name, move_tensor(param, "cpu"))
-    # Freeze model and move to target device
-    freeze(model)
```

Comment:
I'm not sure why freeze(model) was called here. In quantize(), it's called so all of the weights get replaced with their quantized versions, but we're already doing this in requantize() via model.load_state_dict(), so I don't think the freeze(model) call does anything here.